userguide/install.rst

```diff
@@ -999,115 +999,91 @@ option. You can permanently boot from disk by removing the
 VMware ESXi
 ~~~~~~~~~~~
 
-Before using ESXi, read
-`this post
-<https://forums.freenas.org/index.php?threads/sync-writes-or-why-is-my-esxi-nfs-so-slow-and-why-is-iscsi-faster.12506/>`__
-for an explanation of why iSCSI will be faster than NFS.
-
 ESXi is a bare-metal hypervisor architecture created by VMware Inc.
 Commercial and free versions of the VMware vSphere Hypervisor
 operating system (ESXi) are available from the
 `VMware website
-<https://www.vmware.com/products/esxi-and-esx.html?ClickID=bldvs6vzfq1svvmyezgusneeezqfl6qddgne>`__.
-After the operating system is installed on the supported hardware,
-use a web browser to connect to its IP address. The welcome screen
-provides a link to download the VMware vSphere client which is used
-to create and manage virtual machines.
-
-Once the VMware vSphere client is installed, use it to connect to the
-ESXi server. To create a new virtual machine, click
-:menuselection:`File --> New --> Virtual Machine`.
-The New Virtual Machine Wizard will launch as shown in
-:numref:`Figure %s <esxi_new_vm_fig>`.
-
-
-.. _esxi_new_vm_fig:
-
-.. figure:: images/vmware-configuration.png
-
-   New Virtual Machine Wizard
+<https://www.vmware.com/products/esxi-and-esx.html>`__.
 
+When the VMware vSphere client is installed, use it to connect to the
```

Comment:
Can we reword this to avoid the if/then (when)?  Just "Install and use th VMware vSphere client to connect..."